abstract_specification.md

```diff
@@ -1,31 +1,27 @@
 # Introduction
 
-> **Status**: in active development. Version: `0.2.0-dev`
->
-> ℹ️ **Note** - this is an early working draft (WORK IN PROGRESS). It may contain controversial, undecided, or incomplete proposals - feedback is encouraged. It is expect that additional revisions will be required to converge the underlying primitives with use cases, and to incorporate lessons from initial implementation and early users. Please see `issues` and other call-outs throughout the doc for a list of active discussions/debates.
-
 The goal of SOMA (“stack of matrices, annotated”) is a flexible, extensible, and open-source API providing access to annotated, 2D matrix data stored in multiple underlying formats and systems. The vision for this API family includes:
 
-- support access to persistent, cloud-resident datasets,
-- enable use within popular data science environments (e.g., R, Python), using the tools of that environment (e.g., Python Pandas integration),
-- enable "out-of-core" access to data aggregations much larger than single-host main memory
-- enable distributed computation over datasets
-- provide a building block for higher-level API that may embody domain-specific conventions or schema around annotated 2D matrices (e.g., a cell "atlas").
+* support access to persistent, cloud-resident datasets,
+* enable use within popular data-science environments (e.g. R, Python), using the tools of that environment (e.g. Python Pandas integration),
```

Comment:
most of these new hyphens are kind of weird

Reply:
@thetorpedodog : 

* _Man eating shark_ is not the same as _man-eating shark_. (Who's for dinner?) Hyphens save lives.
* Likewise, a _man-eating shark suit_ will not keep you safe but a _man-eating-shark suit_ will. Hyphens save lives.

More info:

* https://en.wikipedia.org/wiki/English_compound#Hyphenated_compound_modifiers

Reply:
That’s true in those cases, but there’s no real risk of ambiguity here; "data (science environment)" is not a reasonable concept, cf:

> The following compound modifiers are not normally hyphenated:
>
> - Compound modifiers that are not hyphenated in the relevant dictionary or that are unambiguous without a hyphen.
> - Where there is no risk of ambiguity:
>   - "a Sunday morning walk"

Reply:
@thetorpedodog with the `-` is better than without -- this is a lot of keystroking we're doing over this `-` ...

Reply:
I disagree; while it’s not necessarily _wrong_, many of the hyphenated constructions read as odd or unnecessary to me. A [cursory Google search](https://www.google.com/search?q=%22data+science+environment%22) shows that a non-hyphenated construction is essentially universal.

Besides, I _love_ making keystrokes.